docs/experiments-gar-t5.md

```diff
@@ -3,33 +3,11 @@
 This guide provides instructions to reproduce the search results of our GAR-T5 model which takes inspiration from the following paper:
 > Mao, Y., He, P., Liu, X., Shen, Y., Gao, J., Han, J., & Chen, W. (2020). [Generation-augmented retrieval for open-domain question answering](https://arxiv.org/abs/2009.08553). arXiv preprint arXiv:2009.08553.
 
-We first need to download the test dataset for evaluation. For both NQ and TriviaQA, there are three types of query generation targets, answer, title and sentence.
-
-## Get the Dataset as tsv
-Download the dataset from HuggingFace and use script to process it to a .tsv file ([TriviaQA](https://huggingface.co/datasets/castorini/triviaqa_gar-t5_expansions) and [NaturalQuestion](https://huggingface.co/datasets/castorini/nq_gar-t5_expansions))
-
-```bash
-export ANSERINI=<path to anserini>
-
-python scripts/gar/query_augmentation_tsv.py \
-  --dataset <nq or trivia> \
-  --data_split <validation or test> \
-  --output_path <default is augmented_topics.tsv> \
-  --sentences <optional> \
-  --titles <optional> \
-  --answers <optional>
-```
-
 ## GAR-T5 enhanced retrieval evaluation
-To evaluate the augmented queries, we need to concatenate and convert them into .tsv format for us to run BM25-search on Pyserini, which is then converted to .json format as required for evaluation.
-
-Without specifying the output path, the default output will be an `augmented_topics.tsv` file in the working directory.
-
-Once we have the tsv file, we can proceed to run search and evaluation
```

Comment:
Perhaps include the first half of this in the end of the README file? Perhaps

## Method 2: Interacting with GAR-T5 Predictions

If you want to generate the pre-built topics yourself from the GAR-T5 Predictions, ... [enter whatever you have in the `Get the download...`